frontend/src/metabase/query_builder/components/view/View.jsx

```diff
@@ -456,7 +455,7 @@ export default class View extends React.Component {
 
     // if we don't have a card at all or no databases then we are initializing, so keep it simple
     if (!card || !databases) {
-      return <LoadingAndErrorWrapper className={fitClassNames} loading />;
```

Comment:
`fitClassNames` came from `fitViewport` which was removed a while back, so it was empty